index.html

```diff
@@ -602,35 +607,22 @@ <h1>Trust Model</h1>
 risk assessment of the verifier.
         </li>
         <li>
-All entities trust the <a>identifier registry</a> to be incorruptible and
-to be a correct record of which identifiers belong to which <a>entities</a>.
+All entities trust the <a>Verifiable data repository</a> to be un-corruptible and
+to be a correct record of which data is owned (and can be updated) by which
+<a>entities</a>.
         </li>
         <li>
-The <a>subject</a> trusts the <a>issuer</a> to issue true
+The <a>subject</a> trusts (since only the holder can verify) the <a>issuer</a> to issue true (i.e., not false)
 <a>credentials</a> about the subject, and to revoke them quickly when
 appropriate.
         </li>
-        <li>
-The <a>holder</a> trusts the <a>repository</a> to store the
-<a>credentials</a> securely, to not release them to anyone other than the
-<a>holder</a>, and to not corrupt or lose them whilst they are in its care.
-        </li>
       </ol>
 
       <p>
-This trust model differentiates itself from other trust models by ensuring that:
+This trust model differentiates itself from other trust models by ensuring that
+the <a>issuer</a> does not need to trust (or even know) the <a>verifier</a>.
       </p>
 
-      <ul>
-        <li>
-The <a>issuer</a> and the <a>verifier</a> do not need to trust the
```

Comment:
This is an important part of the VC Data Model... we can't just remove it.

Reply:
Added back